packages/render-fidelity-tools/src/common.ts

```diff
@@ -237,10 +243,12 @@ export class ImageComparator {
     // when the golden image's background (use top left pixel to represent) is
     // transparent while the candidate(mode-viewer) image is not, it's
     // definitely that case
-    const candidateTopLeftAlpha = candidateImage[3];
-    const goldenTopLeftAlpha = goldenImage[3];
-    if (goldenTopLeftAlpha != candidateTopLeftAlpha) {
-      throw new Error(WARNING_MESSAGE);
+    if (checkSemiTransparent) {
```

Comment:
Some filament's goldens' topleft pixel's alpha is not 0 (probably something i did wrong?), causing a lot of trouble. I just disable this check when comparing model-viewers' goldens with other renderers' golden for now, and i think we should discuss a better solution